my-portfolio/src/components/Background.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from "react";
+import { Row, Col, Nav, Tab } from "react-bootstrap";
+
+class Background extends Component {
```

Comment:
Stateless components can be "PureComponent" of functional components (personal preference)